wellcomecollection/wellcomecollection.org
# Prettier upgrade

## What does this change?

I started getting prettier warning and automatic changes in files, especially around ternaries. This wasn't replicated by @davidpmccormick so I think it had to do with the fact that we're not pinning those packages to a specific version and we might've been using different ones? And possibly something to do with [this change of theirs](https://prettier.io/blog/2023/11/13/curious-ternaries), even though it seems unrelated.

I then upgraded them to their latest version and ran `yarn lint --fix` from the root. Most changes have to do with ternaries, and a few with slight indentation changes.

I like this change, I find it makes complex ones a lot more readable, although like David's said, maybe when they're that complex they should be refactored.

<img width="981" alt="Screenshot 2024-08-07 at 11 17 56" src="https://github.com/user-attachments/assets/bfaa785a-7612-40cc-8928-03982c750ec4">
<img width="1083" alt="Screenshot 2024-08-07 at 11 18 11" src="https://github.com/user-attachments/assets/f8c23d41-f90a-4db6-a374-4a98e9511d05">

This PR is simple but should insure we don't have random conflicts or changes.


## How to test

Run locally
`yarn rm -rf node_modules && yarn rm -rf yarn.lock`
`yarn`
`yarn lint` shouldn't flag anything
Check if you get a different indentation change on a file save. 

## How can we measure success?

Smooth sailing on indents for a bit

## Have we considered potential risks?
N/A
